content/sidebar/agricultural_filial.md

```diff
@@ -0,0 +1,8 @@
+title: О филиале кафедры
+slug: agricultural_presentation_filial
```

Comment:
There should be no `_` in `slug`s! Replace this `slug` with `agricultural-department-branch-presentation`.